sdk/ml/azure-ai-ml/azure/ai/ml/entities/_builders/do_while.py

```diff
@@ -190,16 +190,16 @@ def _validate_port(self, port, node_ports, port_type, yaml_path):
             validation_result.append_error(
                 yaml_path=yaml_path,
                 message=(
-                    f"{port_obj._name} is the {port_type} of {port_obj._owner.name}, "  # pylint: disable=protected-access
+                    f"{port_obj._port_name} is the {port_type} of {port_obj._owner.name}, "  # pylint: disable=protected-access
```

Comment:
Why changed this? what if the output port does not have a port name?

Reply:
I use `_port_name` to replace the previous `_name`. I think currently the output port will always have port name but will not always have name(when we don't register output). Modify the error message in this way will make it the same as previous design.